Sources/Brave/Frontend/Browser/HomePanel/NTPDataSource.swift

```diff
@@ -253,3 +221,20 @@ extension NTPDataSource: PreferencesObserver {
     }
   }
 }
+
+extension NTPSponsoredImageTopSite {
+  var asFavoriteSite: FavoriteSite? {
+    guard let url = destinationURL else {
+      return nil
+    }
+    return FavoriteSite(url, name)
+  }
+}
+
+extension NTPBackgroundImage {
+  static let fallback: NTPBackgroundImage = .init(
+    imagePath: Bundle.module.url(forResource: "corwin-prescott-3", withExtension: "jpg")!,
+    author: "Corwin Prescott",
+    link: URL(string: "https://www.brave.com")!
```

Comment:
is this URL correct? I see in the past for this author we used nothing, empty string

Reply:
Yeah this is what comes back in the json downloaded from the component updater. This exact background will essentially be redownloaded and have the exact same link